CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contribution Guidelines
+
+Hello. 👋
+
+Welcome to the newest project for FreeCodeCamp. We are glad you are here!
+
+We strictly enforce our ["Code of Conduct"](https://www.freecodecamp.org/code-of-conduct). Please take a moment to read it. It's only 196 words long.
+
+Happy contributing.
+
+## We are just getting started
+
+Implementation details are still being discussed and the project is currently at a very early stage.  The stage of the project is changing on a daily basis, thoughts and ideas are being contributed at a very high pace.
+
+## What do we need help with right now?
+
+We are in the very early stages of development on this new application. We value your insight and expertise.  In order to prevent duplicate issues, please search through our existing issues to see if there is one for which you would like to provide feedback. We are currently trying to consolidate many of the issues based on topics (Docs, UI/UX, API architecture, Tech Stack, etc)
+
+## How to submit an issue
+
+- Search through current issues to verify that you are not creating a duplicate issue.
+- Use the issue template as a guide to tell us more about your issue:
```

Comment:
Couldn't [GitHub issue templates](https://help.github.com/en/articles/creating-issue-templates-for-your-repository) be used for this?